data/presets/amenity/post_box/post_box_gb.json

```
@@ -0,0 +1,45 @@
+{
+    "icon": "temaki-post_box",
+    "locationSet": {
+        "include": [
+            "gb"
+        ]
+    },
+    "fields": [
+        "operator",
+        "collection_times",
+        "post_box:type",
+        "royal_cypher",
+        "ref"
+    ],
+    "moreFields": [
+        "colour",
+        "indoor",
+        "level",
+        "material",
+        "support",
+        "wheelchair"
+    ],
```

fields can also be referenced between presets, in order to reduce duplication:

```suggestion
    "moreFields": [
        "{amenity/post_box}",
        "drive_through"
    ],
```

---

> PS: I included `drive_through` here, as it is uses in small quantities even in the UK ([example](https://www.openstreetmap.org/node/5087338620)). Even if these were all mapping mistakes, keeping this fields as an optional field will at least not hide those potential errors in the data.

There's less than 60 of them in the UK and I've only found one other example so far of a "drive through" post box, as in one that is close enough to the road that you don't have to leave your vehicle. [example of one that was wrongly tagged as yes](https://www.bing.com/maps?cp=51.389306%7E-0.141818&lvl=17.0&v=2&sV=1&style=x&dir=87.6&pi=-8.7)

Most of them are probably tagging errors, especially as you must click twice on drive through check box to get the value of "no". So if it has to be included in this preset then it should be below all of the other fields.

---

yeah, usage is really pretty niche in the UK (and many other countries[^1]).

It's just that including it makes maintaining these regional presets so much easier if the `moreFields` are simply _"just everything this preset could ever have, even if unlikely in a given region"_. IMHO, It also doesn't hurt too much from a user experience point of view to include them, because the optional fields are only added if a user actually searches for it or the object already has a matching tag.

[^1]: I'll probably change the default preset to not include the "drive_through" field by default and create a separate regional preset for countries like the US where such post boxes are found more commonly.